client/state/checklist/reducer.js

```diff
@@ -17,10 +17,9 @@ import { items as itemSchemas } from './schema';
 
 const setChecklistTaskCompletion = ( state, taskId, completed ) => ( {
 	...state,
-	tasks: {
-		...state.tasks,
-		[ taskId ]: { ...get( state.tasks, [ taskId ] ), completed },
-	},
+	tasks: state.tasks.map( task =>
```

Comment:
Any reason not to use the native `Array.prototype.map()`?

Reply:
We use map here, but I understand this question was meant for line 42? In this case - we're using lodash `map` because the API returns an object that we need to turn into an array.

Reply:
Sorry. I got crossed-eyes. Please ignore me.